core/app/models/spree/tax/application.rb

```diff
@@ -0,0 +1,51 @@
+module Spree
+  module Tax
+    class Application
```

Comment:
`Adjuster`, `Performer`, `Applier`, `Applicator`, `OrderTaxer`. Just throwing in ideas here, I second Johns thought.